src/main/java/org/rulez/demokracia/pdengine/IVoteManager.java

```diff
@@ -25,7 +25,7 @@ VoteAdminInfo createVote(String voteName, Set<String> neededAssurances, Set<Stri
 
 	String addChoice(String adminKey, String voteId, String choiceName, String user);
 	
-	String deleteChoice(String voteId, String choiceId, String adminKey) throws ReportedException;
```

Comment:
https://adadocs.demokracia.rulez.org/PDEngine/edemo/master/index.html#3044c21c-7606-4098-8176-9dd5b93c20ad

Reply:
The modify choice has the same in the Deviaton Model (I mean the return "OK") but it is void in this interface.
So which is the good?

Reply:
The modify choice has the same in the Deviaton Model (I mean the return "OK") but it is void in this interface.
So which is the good?

Reply:
This diff made it void. It should be String.